frameworks/CSharp/appmpower/appmpower.dockerfile

```
@@ -0,0 +1,22 @@
+FROM mcr.microsoft.com/dotnet/sdk:5.0 AS build
+RUN apt-get update
+RUN apt-get -yqq install clang zlib1g-dev libkrb5-dev libtinfo5
+
+WORKDIR /app
+COPY src .
+RUN dotnet publish -c Release -o out -r linux-x64
+
+FROM mcr.microsoft.com/dotnet/aspnet:5.0 AS runtime
```

If you did switch to .NET 6.0; up to you, would need to also change this line
```suggestion
FROM mcr.microsoft.com/dotnet/aspnet:6.0 AS runtime
```